src/login/login.component.tsx

```diff
@@ -22,15 +22,14 @@ export default function Login(props: LoginProps) {
         includeAuthStatus: true
       }).subscribe(
         authResult => {
+          setCheckingIfLogged(false);
           if (authResult.authenticated) {
             props.history.push("/home");
           }
         },
         err => {
-          throw err;
-        },
-        () => {
```

Comment:
This is a "completed" function, which fires whenever the observable is finished producing values. In the case of getCurrentUser(), the observable never completes because the current user could always change in the future. The fix is to move it the "next" and "error" functions so that it gets run whenever we get a value from the api